src/__tests__/Search/SelectedRevision.test.tsx

```diff
@@ -46,30 +48,21 @@ describe('SelectedRevision', () => {
     const searchInput = screen.getAllByRole('textbox')[0];
     await user.click(searchInput);
 
-    await user.click(screen.getAllByTestId('checkbox-0')[0]);
+    const noArmsLeft = await screen.findByRole('button', {
+      name: /you've got no arms left!/,
+    });
+    const noArmsLeftCheckbox = within(noArmsLeft).getByRole('checkbox');
 
-    expect(
-      screen.getAllByTestId('checkbox-0')[0].classList.contains('Mui-checked'),
-    ).toBe(true);
+    await user.click(noArmsLeft);
+    expect(noArmsLeft).toHaveClass('item-selected');
+    expect(noArmsLeftCheckbox).toBeChecked();
+    expect(noArmsLeft.querySelector('.Mui-checked')).toBeInTheDocument();
 
-    expect(
-      screen.getAllByTestId('selected-revs-search')[0],
-    ).toBeInTheDocument();
-    expect(document.body).toMatchSnapshot();
-    await act(async () => void jest.runOnlyPendingTimers());
+    const selectedRevsContainer = screen.getByTestId('selected-revs-search');
+    expect(selectedRevsContainer).toMatchSnapshot();
```

Comment:
here I changed the snapshotted element (from `document.body` to `selectedRevsContainer`). This makes a much smaller snapshot (easier to read, faster). I'm considering doing the same in other snapshots... but maybe later.